src/index.html

```diff
@@ -16,16 +17,19 @@
       href="style.css"
     />
   </head>
+
   <body>
     <input
+      class="search-bars search-bars-1"
       type="text"
-      data-qa="keypress"
+      data-qa="big"
       placeholder="Try “Los Angeles“"
     />
```

Comment:
The 'input' elements are missing the 'name' attribute. This attribute is important for form submission and accessibility.